Compute/tests/System/V1/SmokeTest.php

```diff
@@ -87,30 +104,63 @@ public function testInsertInstance()
             self::$projectId,
             self::ZONE
         );
-        try{
-            $operationClient = new ZoneOperationsClient();
-            while (true) {
-                $op = $operationClient->get(
-                    $operation->getName(),
-                    self::$projectId, self::ZONE
-                );
-                $status = $op->getStatus();
-                if (in_array($status, [Status::DONE, Status::UNDEFINED_STATUS])) {
-                    break;
-                }
-            }
-            $instance = self::$instancesClient->get(
-                $name,
-                self::$projectId,
-                self::ZONE
-            );
-        } finally {
-            self::$instancesClient->delete($name, self::$projectId, self::ZONE);
-        }
-        self::assertEquals($name, $instance->getName());
+        $this->waitForZonalOp($operation);
+        self::$dirty = true;
+    }
+
+    public function testInsertInstance()
+    {
+        $this->insertInstance();
+        $instance = self::$instancesClient->get(
+            self::$name,
+            self::$projectId,
+            self::ZONE
+        );
+        self::assertEquals(self::$name, $instance->getName());
         self::assertEquals(self::$machineType, $instance->getMachineType());
     }
 
+    public function testPatchInstance(){
```

Comment:
Consistency with code styles
```suggestion
    public function testPatchInstance()
    {
```